### PR TITLE
Get all constant values to a identical type, use static methods (close #111)

### LINKED ISSUE #111: Get all constant values to a identical type

### PR DESCRIPTION
**Pull request by churkin**
\сс @inikulin @LavrovArtem @miherlosev

### CODE REVIEW

File: src/const.js [hunk 1]
```diff
@@ -5,24 +5,22 @@
...
 
-CONST.UPLOAD_SANDBOX_HIDDEN_INPUT_NAME = 'upload-89fc3-info';
+CONST.UPLOAD_SANDBOX_HIDDEN_INPUT_NAME = '@hammerhead|upload-info-hidden-input@';
 
-CONST.HOVER_PSEUDO_CLASS_ATTR = 'data-1b082a6cec-hover';
+CONST.HOVER_PSEUDO_CLASS_ATTR = 'data-hammerhead-hovered';
 
 CONST.XHR_CORS_SUPPORTED_FLAG   = 0x10;
-CONST.XHR_REQUEST_MARKER_HEADER = 'x-tc-xm-cd46977f';
+CONST.XHR_REQUEST_MARKER_HEADER = '%hammerhead|xhr-request-marker-header%';
```
Comment (inikulin) [inline src/const.js]: Why not `@` prefix and postfix here?

File: src/const.js [hunk 2]
```diff
@@ -5,24 +5,22 @@
 
 var CONST = {};
 
-CONST.DOM_SANDBOX_OVERRIDE_DOM_METHOD_NAME = 'tc_odm_e01a2ff5';
-CONST.DOM_SANDBOX_PROCESSED_CONTEXT        = 'tc-spc-c1208da2';
-CONST.DOM_SANDBOX_STORED_ATTR_POSTFIX      = '_e01a2f_stored';
-CONST.EVENT_SANDBOX_WHICH_PROPERTY_WRAPPER = 'tc-e01a2f-which';
+CONST.DOM_SANDBOX_OVERRIDE_DOM_METHOD_NAME = '@hammerhead-override-dom@';
```
Comment (inikulin) [inline src/const.js]: What is the purpose of this constant? It's naming is quite unclear

Comment (inikulin) [inline src/const.js]: And why it not follows format `@hammerhead|...@`?

File: src/const.js (hunk 1 above)
Comment (churkin) [inline src/const.js]: We can not use '%' because the const values used in the url attributes and can not be decoded/ecoded.

Comment (inikulin) [inline src/const.js]: You mean `@`? It's used in HTTP header name and not URL AFAIK and I see that `@` is disallowed there: http://www.w3.org/Protocols/rfc2616/rfc2616-sec2.html#sec2.2
We can get rid of prefix and postfixes then and just keep name it `hammerhead|xhr-request-marker-header`. What do you think?

File: src/const.js (hunk 2 above)
Comment (churkin) [inline src/const.js]: uses in the regex

Comment (inikulin) [inline src/const.js]: @churkin What do you mean?

**Comment by inikulin** [comment]
tests fail

**Comment by inikulin** [comment]
BTW what about #112? Will this PR close it as well?

**Comment by churkin** [comment]
I'll close #112. We decided not to do it because of the inconvenience of use

**Comment by inikulin** [comment]
ok

**Comment by inikulin** [comment]
Requires rebase now

**Comment by inikulin** [comment]
@churkin Seems like it really hangs up in IE 10 and 9

**Comment by churkin** [comment]
FPR

**Comment by miherlosev** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Pull request merged**